src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+	
+	public abstract String getPrintableString();
```

Comment:
This method definition creates a lot of confusion. It is missing head comments as well as overloading the required function.